projects/core/src/product/occ/product-reviews.service.ts

```diff
@@ -0,0 +1,61 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+import { Observable, throwError } from 'rxjs';
+import { catchError } from 'rxjs/operators';
+
+import { ReviewList, Review } from '../../occ/occ-models/occ.models';
+import { DynamicTemplate } from '../../config/utils/dynamic-template';
+
+import { OccProductConfig } from './product-config';
+import { ProductOccService } from './product-occ.service';
+
+@Injectable()
+export class OccProductReviewsService extends ProductOccService {
+  constructor(
+    private http: HttpClient,
+    private config: OccProductConfig,
+    private dynamicTemplate: DynamicTemplate
```

Comment:
While I understand I made the suggestion to make this an injected class, I regret the fact that the constructor gets more polluted now. What about going back to static functions? ;) What's you opinion?

Reply:
I did prefer the static functions to be honest. I'll go back to that method.